src/app/scheme/device-item-group/device-item-group.component.html

```diff
@@ -3,10 +3,9 @@
      [ngStyle]="{'border-color': group.status_info?.color}">
     <div class="dig-header">
         <div class="invisible-anchor" id="scheme-group-{{group.id}}" ></div>
-        <a (click)="openParamsDialog(group.id)" *ngIf="group?.params?.length > 0"><!--  *ngIf="group.params.length !== 0" -->
+        <a (click)="openParamsDialog(group.id)">
             <h4>{{group.title ? group.title : group.type.title}}</h4>
         </a>
```

Comment:
Ты уверен что нужно было не так?
```suggestion
        <a (click)="openParamsDialog(group.id) *ngIf="group?.params?.length > 0 || editorModeEnabled">
            <h4>{{group.title ? group.title : group.type.title}}</h4>
        </a>
        <h4 *ngIf="group.params.length === 0 && !editorModeEnabled">{{group.title ? group.title : group.type.title}}</h4>
```